### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by davie-robertson**
fixed the formatting for Events

### CODE REVIEW

File: packages/select/README.md [hunk 1]
```diff
@@ -253,24 +253,8 @@ Name                                          | Description
...
-:            :                      :                   : `Set<number>` if     :
-:            :                      :                   : multi and `number`   :
-:            :                      :                   : if single), and      :
-:            :                      :                   : `diff` (of type      :
-:            :                      :                   : `IndexDiff`**)       :
-:            :                      :                   : represents the diff  :
-:            :                      :                   : of added and removed :
-:            :                      :                   : indices from         :
-:            :                      :                   : previous selection.  :
+| `action`   | `mwc-list`           | `ActionDetail`*   | Fired when a selection has been made via click or keyboard aciton.|
```
Comment (asyncLiz) [inline packages/select/README.md]: Misspelled action and there should be a space between the period and the edge of the table
```suggestion
| `action`   | `mwc-list`           | `ActionDetail`*   | Fired when a selection has been made via click or keyboard action. |
```

File: packages/select/README.md [hunk 2]
```diff
@@ -253,24 +253,8 @@ Name                                          | Description
...
+| `selected` | `mwc-list`           | `SelectedDetail`* | Fired when a selection has been made. `index` is the selected index (will  be of type `Set<number>` if multi and `number` if single), and `diff` (of type `IndexDiff`**) represents the diff of added and removed indices from previous selection.|
```
Comment (asyncLiz) [inline packages/select/README.md]: Also a space here!

```suggestion
| `selected` | `mwc-list`           | `SelectedDetail`* | Fired when a selection has been made. `index` is the selected index (will  be of type `Set<number>` if multi and `number` if single), and `diff` (of type `IndexDiff`**) represents the diff of added and removed indices from previous selection. |
```

**Comment by asyncLiz** [comment]
Apparently I've never signed a CLA with my personal email address that got used in the co-author commits 😂

**Comment by davie-robertson** [comment]
> Looks good! By the way, you can click on the refresh icon next to the reviewer's name to let them know when you made updates so they can re-review it!

thanks, I didn't know that

**Pull request merged**